auction9-frontend/src/app/add-update-auction-dialog/add-update-auction-dialog.component.ts

```diff
@@ -161,6 +161,7 @@ export class AddUpdateAuctionDialogComponent implements OnInit {
               panelClass: ['light-snackbar']
             });
         }
+      setTimeout(() => location.reload(), 800);
```

Comment:
What's this for?

Reply:
I used it to refresh page and see changed results

Reply:
You can't be sure if 800ms will be enough. Also reloading the page is not the way SPA handles data changes.

Proper way to do it is to have API return updated resource and use that data to update the view.

Reply:
I agree, now it's removed. Better way will be implemented soon